pkgs/development/python-modules/moderngl_window/default.nix

```diff
@@ -28,11 +29,14 @@ buildPythonPackage rec {
   # Tests need a display to run.
   doCheck = false;
 
+  pythonImportsCheck = [ "moderngl_window" ];
+
   meta = with lib; {
     homepage = "https://github.com/moderngl/moderngl_window";
     description = "Cross platform helper library for ModernGL making window creation and resource loading simple";
     license = licenses.mit;
-    platforms = platforms.linux; # should be mesaPlatforms, darwin build breaks.
+    broken = stdenv.isDarwin; # darwin build breaks
```

Comment:
I will check and submit a followup PR if I can fix it. I would really prefer to merge this now because moderngl is actively broken in certain scenarios due to this update missing.